### PR TITLE
feat(console): update page titles in pezzo consoles 

### LINKED ISSUE #192: Update titles in the Pezzo Console
### Proposal

Currently, all Console pages have the `Pezzo` title. This feature is about adding a title for all pages. The format should be as follows: `{feature} - Pezzo`.

Examples:
- Login page: `Login - Pezzo`
- Project Dashboard page: `Dashboard - Pezzo`
- Prompts page: `Prompts - Pezzo`
- Environments page: `Environments - Pezzo`
- ...etc

### Use-Case

This will provide a better user experience, and make it much easier to understand which Pezzo tab is most relevant by providing context at a glance.

### How to approach this

We have `usehooks-ts` nstalled. More specifically, you can use the [useDocumntTitle](https://usehooks-ts.com/react-hook/use-document-title) hook to change the title per page.

All pages can be found in a component with a `*Page.tsx` file name. E.g. `LoginPage.tsx`, `ProjectsPage.tsx` and so on.

### PR DESCRIPTION
**Pull request by senderh55**
fixes #192

### CODE REVIEW

File: apps/console/src/app/pages/InfoPage.tsx [hunk 1]
```diff
@@ -1,8 +1,10 @@
 import { version } from "@pezzo/common";
 import { Typography } from "antd";
 import { InlineCodeSnippet } from "../components/common/InlineCodeSnippet";
+import { usePageTitle } from "../lib/hooks/usePageTitle";
 
 export const InfoPage = () => {
+  usePageTitle("Login");
```
Comment (arielweinberger) [inline apps/console/src/app/pages/InfoPage.tsx:7]:
```suggestion
  usePageTitle("Info");
```

File: apps/console/src/app/pages/organizations/OrgPage.tsx [hunk 2]
```diff
@@ -83,7 +84,7 @@ export const OrgPage = () => {
       ),
     [tabsItems, isOrgAdmin]
   );
-
+  usePageTitle(`${organization?.name || "Organization"}`);
```
Comment (arielweinberger) [inline apps/console/src/app/pages/organizations/OrgPage.tsx:87]:
```suggestion
  usePageTitle(organization?.name);
```
With the fallback we don't need the `||` here.

File: apps/console/src/app/pages/organizations/onboarding/OnboardingPage.tsx [hunk 3]
```diff
@@ -39,7 +40,7 @@ export const OnboardingPage = () => {
     useUpdateCurrentUserMutation();
   const { mutateAsync: createProject, isLoading: isProjectCreationLoading } =
     useCreateProjectMutation();
-
+  usePageTitle(`Onboarding ${organization?.name || ""}`);
```
Comment (arielweinberger) [inline apps/console/src/app/pages/organizations/onboarding/OnboardingPage.tsx:43]:
```suggestion
  usePageTitle("Onboarding");
```
I think this is enough.

File: apps/console/src/app/pages/projects/ProjectPage.tsx [hunk 4]
```diff
@@ -6,10 +6,12 @@ import { useState } from "react";
 import { GettingStartedWizard } from "../../components/getting-started-wizard/GettingStartedWizard";
 import { GettingStartedWizardProvider } from "../../lib/providers/GettingStartedWizardProvider";
 import { Loader } from "../../components/common/Loader";
+import { usePageTitle } from "../../lib/hooks/usePageTitle";
 
 export const ProjectPage = () => {
   const { project, isLoading } = useCurrentProject();
   const [clicked, setClicked] = useState(false);
+  usePageTitle(project?.name || "Project");
```
Comment (arielweinberger) [inline apps/console/src/app/pages/projects/ProjectPage.tsx:14]:
```suggestion
  usePageTitle(project?.name);
```
With the fallback we don't need the `||`

File: apps/console/src/app/pages/prompts/PromptPage.tsx [hunk 5]
```diff
@@ -24,7 +25,7 @@ const TabLabel = styled.div`
 export const PromptPage = () => {
   const { prompt, isLoading } = useCurrentPrompt();
   const [activeView, setActiveView] = useState("edit");
-
+  usePageTitle(prompt?.name || "Prompt");
```
Comment (arielweinberger) [inline apps/console/src/app/pages/prompts/PromptPage.tsx:28]:
```suggestion
  usePageTitle(prompt?.name);
```

File: apps/console/src/app/pages/prompts/PromptsPage.tsx [hunk 6]
```diff
@@ -22,6 +23,7 @@ export const PromptsPage = () => {
       gqlClient.request(GET_ALL_PROMPTS, { data: { projectId: project?.id } }),
     enabled: !!project?.id,
   });
+  usePageTitle(`${project.name || ""} prompts`);
```
Comment (arielweinberger) [inline apps/console/src/app/pages/prompts/PromptsPage.tsx:26]:
```suggestion
  usePageTitle(project?.name && `${project.name} Prompts`);
```

**Review by arielweinberger — CHANGES_REQUESTED** [review]
Looks great overall! Let's try the changes I've suggested.

**Review by senderh55 — COMMENTED** [review]
Thanks!

**Comment by arielweinberger** [comment]
Tested, seems to work very well. Well done, @senderh55

**Review by arielweinberger — APPROVED** [review]

**Pull request merged**